app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_progress.html

```diff
@@ -26,33 +27,33 @@ <h2>Users Progress for {{ reader_study.title }}</h2>
 
     <ul class="list-group list-group-flush">
         {% for user in users %}
-            <li class="list-group-item">
+            <li class="list-group-item mb-1">
                 <div class="row">
-                    <div class="col-3 mb-1">
+                    <div class="col-6 col-lg-2 col-md-3 p-1">
                         <div class="h-100 d-flex justify-content-start align-items-center">
                             {{ user.obj|user_profile_link }}
                         </div>
                     </div>
-                    <div class="col-4 mb-1">
-                        <div class="h-100 d-flex align-items-center">
-                            <div class="flex-fill progress h-100">
-                                <div class="progress-bar" role="progressbar"
-                                     style="width: {{ user.progress.questions }}%"
-                                     aria-valuenow="{{ user.progress.questions }}"
-                                     aria-valuemin="0" aria-valuemax="100">
-                                </div>
+                    <div class="col-6 col-lg-4 col-md-3 p-1">
+                        <div class="progress h-100">
+                            <div class="progress-bar" role="progressbar"
+                                 style="width: {{ user.progress.questions }}%"
+                                 aria-valuenow="{{ user.progress.questions }}"
+                                 aria-valuemin="0" aria-valuemax="100">
+                                {{ user.progress.questions|floatformat }}%
                             </div>
                         </div>
                     </div>
-                    <div class="col-2 mb-1">
-                        <div class="h-100 d-flex align-items-center">
-                            <div>
-                                &nbsp;Progress: {{ user.progress.questions|floatformat }}%
-                            </div>
+                    <div class="col-6 col-md-3 p-1">
```

Comment:
Yes, if you want to hide this you can add a feature flag. Make a new boolean setting which is named consistently with the existing reader study settings that we can control with environment variables, pass this through to the template, default false.